cmd/server_test.go

```diff
@@ -0,0 +1,134 @@
+package cmd_test
```

Comment:
For completeness, could you add tests for `ServerDelete` and `ServerLs`?

Reply:
Sure thing. I didn't want to clutter too much the PR but I'm happy to add some more test. :smile:

Reply:
Ah, IIRC there was a small issue for testing this. I will need to add a method to the TabWriter, to set and override the Writer.

I think it's something useful anyway, if you want to add the opportunity to set the output to a file, for example.

Reply:
Tests for `serverDelete` https://github.com/rancher/cli/pull/349/commits/52441bfdeb783f91c0b5740e315bd044159ea8fd that led to a small fix (`CurrentServer` was kept also after a delete).

And `serverLs` https://github.com/rancher/cli/pull/349/commits/8349461de9f339c709a41a6c3b56db82421e66be

As you can see I had to create a new func to create a TableWriter, to pass just the needed configuration. Now the `io.Writer` can be customized (and used in out tests), also it accept the `quiet` and `format` options.